components/Info/Info.tsx

```diff
@@ -0,0 +1,20 @@
+import styles from "../Info/Info.module.css";
+export type InfoProps = {
+  firstWord: string;
+  otherWords: string;
+  information: string;
+};
+
+function Info({ firstWord, otherWords, information }: InfoProps) {
+  return (
+    <div className={styles.container}>
+      <div className={styles.firstWord}>
+        {firstWord}
+        <span className={styles.secondWord}>{otherWords}</span>
```

Comment:
Naming `firstWord` and `otherWords` seems a bit weird.
Maybe there's a better way to name these?